src/resources/views/crud/buttons/delete.blade.php

```diff
@@ -47,9 +47,9 @@ className: "bg-danger",
 			          	// Show an error alert
 			              swal({
 			              	title: "{!! trans('backpack::crud.delete_confirmation_not_title') !!}",
-			              	text: "{!! trans('backpack::crud.delete_confirmation_not_message') !!}",
+                            text: result['error'] ? result['error'] : "{!! trans('backpack::crud.delete_confirmation_not_message') !!}",
 			              	icon: "error",
-			              	timer: 2000,
+			              	timer: 4000,
```

Comment:
Also made the duration the same as the others. Nice catch!

Reply:
What do you think about making timer value configurable?